huxley/www/static/js/huxley/components/AdvisorAssignmentsView.js

```diff
@@ -25,7 +25,7 @@ var AdvisorAssignmentsView = React.createClass({
   ],
 
   contextTypes: {
-    session: React.PropTypes.number
+    conference: React.PropTypes.object
```

Comment:
check out `PropTypes.shape`. It allows you to specify the types of each property of the object: https://facebook.github.io/react/docs/reusable-components.html#prop-validation

You can define the proptypes in another module (e.g. ConferenceContext) and `require` them so you don't have to write out the entire shape everywhere you have to write `contextTypes` for a component.